python-example-elasticsearch-extension/README.md

```diff
@@ -1,70 +1,41 @@
-# Example ElasticSearch Extension in Python
-The provided code sample demonstrates how to get a basic extension written in Python 3 up and running.
+# Example Logs API Extension in Python for Elasticsearch
+The provided code sample demonstrates how to get a basic Logs API extension for Elasticsearch written in Python 3 up and running.
 
-> Note: This extension requires the Python 3 runtime to be present in the Lambda execution environment of your function.
+> Note: This extension requires the Python 3 runtime to be present in the Lambda execution environment of your function. This example code is not production ready. Use it with your own discretion after testing thoroughly.
 
-There are two components to this sample:
-* `extensions/`: This sub-directory should be extracted to /opt/extensions where the Lambda platform will scan for executables to launch extensions
-* `python-example-elasticsearch-extension/`: This sub-directory should be extracted to /opt/python-example-extension which is referenced by the `extensions/python-example-elasticsearch-extension` executable and includes a Python executable along with all of its necessary dependencies.
+In this example, we start by developing a simple extension and then add the ability to read logs from the Logs API. For more details on building an extension, please read the Extension API Developer Guide.
 
-## Prep Python Dependencies
-Install the extension dependencies locally, which will be mounted along with the extension code.
-
-```bash
-$ cd python-example-elasticsearch-extension
-$ chmod +x extension.py
-$ pip3 install -r requirements.txt -t .
```

Comment:
There will be a need to pip install requirements, right?

Reply:
Yes it requires the requests module.

Reply:
Removed the dependency on the requests module.